config/prow/cluster/gcsweb_ingress.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: gcsweb
+  namespace: gcsweb
+  labels:
+    app: gcsweb
+  annotations:
+    cert.gardener.cloud/issuer: ci-issuer
+    cert.gardener.cloud/purpose: managed
+    dns.gardener.cloud/class: garden
+    dns.gardener.cloud/dnsnames: gcsweb.gardener.cloud
```

Comment:
Should we rather call this `gcsweb.prow.gardener.cloud`? (same below)

Reply:
Actually, this is not prow-specific and rather a general GCS bucket browser. We could potentially use it for other things as well later on.
We were thinking about `gcsweb.prow.gardener.cloud` as well, but decided for `gcsweb.gardener.cloud` in the end, because of this and symmetry with `gcsweb.k8s.io`.
WDYT?

Reply:
OK, understood. Then let's keep it as suggest. Thanks for explaining!